modules/dxp/apps/commerce/commerce-application-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -143,8 +143,10 @@ renderResponse.setTitle(LanguageUtil.get(request, "applications"));
 		<portlet:param name="redirect" value="<%= currentURL %>" />
 	</portlet:actionURL>
 
-	<aui:script require="metal-dom/src/all/dom as dom,frontend-js-web/liferay/modal/commands/OpenSimpleInputModal.es as modalCommands">
-		var handleAddApplicationBrandButtonClick = dom.delegate(
+	<aui:script require="frontend-js-web/liferay/modal/commands/OpenSimpleInputModal.es as modalCommands,frontend-js-web/liferay/delegate/delegate.es as delegateModule">
```

Comment:
Same as on the other PR, I'd sort this before `ci:forward`, to reduce the chances of it getting rejected.